src/components/CookieLaw.vue

```diff
@@ -73,16 +75,39 @@
       }
     },
     created () {
+      // Check for availability of localStorage
+      try {
+        const test = '__vue-cookielaw-check-localStorage'
+
+        window.localStorage.setItem(test, test)
+        window.localStorage.removeItem(test)
+      } catch (e) {
+        console.error('Local storage is not supported, falling back to cookie use')
+        this.supportsLocalStorage = false
+      }
+
       if (!this.getVisited() === true) {
         this.isOpen = true
       }
     },
     methods: {
       setVisited () {
-        localStorage.setItem('cookie:accepted', true)
+        if (this.supportsLocalStorage) {
+          localStorage.setItem('cookie:accepted', true)
+          console.log('Set using localStorage')
+        } else {
+          Cookie.set('cookie:accepted', true)
+          console.log('Set using cookie')
```

Comment:
Can you please remove the console.logs() ? They should not end up in production.